### ISSUE TITLE
PROJECT IS DEAD

### ISSUE THREAD

**Issue by xtrasmal** [mprzytulski/phpstorm-behat#42]
-- we need to do it ourselves

**Comment by keithdtyler** [comment]
Honestly phpstorm 8's builtin behat support is actually quite good so it seems like this is less needed anyway.

**Comment by xtrasmal** [comment]
true. The author of this plugin also holds the phpspec plugin. Too bad that is not included in phpstorm yet.
